homework/H2_Linear_Algebra.jl

```diff
@@ -731,8 +732,123 @@ md"How does a high/low bandwidth affect the prediction? How does regularization
 You can also implement [more kernel functions](https://en.wikipedia.org/wiki/Positive-definite_kernel#Examples_of_p.d._kernels) and pass them to the `kernel_ridge` keyword-argument `kernel` to see how the prediction changes.
 "
 
+# ╔═╡ 71c2b3dd-3fb4-4f8b-9e19-a043e05ff498
+md"### Exercise 4 – Multiple dispatch on diagonal matrix"
+
+# ╔═╡ c986580b-fe8c-4b68-9fa1-230b429e6de8
+md"""
+### Overview diagonal matrix
+A diagonal matrix is one that has non-zero elements only in the diagonal.
+
+An example for a diagonal matrix would be:
+"""
+
+# ╔═╡ 57d23d48-4f7b-434c-95f3-dd6c704dbb0b
+A₁ = [3 0 0 
+	  0 7 0 
+	  0 0 -4]
+
```

Comment:
Could you fun JuliaFormatter via `format(".")` or by using the matching VSCode command?

Reply:
I have run `format()` on the second notebook. And I must admit that I dont like how this is visualized now in the notebook
![image](https://github.com/user-attachments/assets/b6f15fe1-0b56-41c2-b41e-83cf7481ffcb)
I suppose I will instead write this definition as `A₁ = [3 0 0; 0 7 0; 0 0 -4]`